### ISSUE TITLE
Virtual stick commands occasionally unresponsive on Mavic 2 Pro

### ISSUE THREAD

**Issue by waldron87** [dji-sdk/Mobile-SDK-Android#1239]
We are having sporadic virtual stick command issues with MSDK V4.16.4 when used with aircraft firmware 01.00.0670 and RC firmware 01.00.0640. Aircraft is Mavic 2 Pro and we are using in combination with Odroid N2.

The issue is that the aircraft sporadically fails to respond to pitch or roll commands after auto takeoff has completed, but throttle command does appear to remain responsive. All prerequisite calls are made and are reported successful before we start sending the virtual stick commands, and we know there is no problem with our workflow because 99% of the time, the drone is responsive to all of our virtual stick commands.

It is an indoor use case, so no/limited access to GPS. RC flight mode switch is always in 'P' mode and we always get the callback to say that the stick commands are successful, but the drone simply doesn't respond.

Are you able to support with this issue please?

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/87195'>Zendesk ticket #87195</a>:<br><br>What is the frequency of occurrence of this issue? Are there any obstacles near the aircraft when this issue occurs?
<br><p><em><small>°°°</small></em></p>

**Comment by waldron87** [comment]
This ticket is a duplicate of http://bitly.ws/LNrV. Are you happy to continue communications via that channel?

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/87195'>Zendesk ticket #87195</a>:<br><br>Of course, I can do that. I will reply on the ticket  you provided.
<br><p><em><small>°°°</small></em></p>
